core/client/lemverse.hbs.html

```diff
@@ -21,7 +21,11 @@
       {{/each}}
     </div>
 
+    {{#if not onboarded}}
+    {{> onboarding}}
+    {{else}}
```

Comment:
Is that the only mechanism to block the user from entering the level as a non-guest?

If so, I guess we're still spawning the character in the level, and I don't think that's right.

Reply:
Actually, the spawning character in the level occurs when ever you arrive on lemverse, even if your not already logged. The onboarding appear when you create a new registration or log with an account who is unboarded. 

If I go in your way, this will be weird to spawn character -> create new account then remove spawn character -> finish onboarding then respawn the character

Reply:
no, just keep the user as a guest until the onboarding is over